tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Handles logging functionality.
```

Comment:
I feel we can modify the event class only and put this logic in that. We will have to call the event class only then.